libraries/OPAMP/library.properties

```diff
@@ -0,0 +1,10 @@
+name=OPAMP
+version=1.0.0
+author=Maximilian Gerhardt
+maintainer=Maximilian Gerhardt <maximilian.gerhardt@rub.de>
+sentence=Library for using the OPAMP on Arduino Uno R4 boards.
+paragraph=Supports OPAMP on Arduino Uno R4 (Minima, WiFI).
+category=Signal Input/Output
+url=https://github.com/arduino/ArduinoCore-renesas/tree/master/libraries/OPAMP

```

Comment:
```suggestion
url=https://github.com/arduino/ArduinoCore-renesas/tree/main/libraries/OPAMP
```

Reply:
Actually I copied this from the WiFi library, so that's wrong there too-

https://github.com/arduino/ArduinoCore-renesas/blob/1c9b08697388a887d3a3114704661fdca0b4948c/libraries/WiFi/library.properties#L8